types.go

```diff
@@ -450,6 +450,7 @@ type PersonalizationResponse struct {
 	Limit    int                      `json:"limit"`
 	Offset   int                      `json:"offset"`
 	Version  string                   `json:"version"`
+	Next     string                   `json:"next"`
```

Comment:
can you add a `parseNext()` method to `PersonalizationResponse`, similar to the ones in `readResponse` and `filterResponse` ?